app/Services/Logging/ProductionRequestLogger.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Services\Logging;
+
+use Illuminate\Http\Request;
+
+class ProductionRequestLogger extends AbstractRequestLogger
+{
+    protected function prepareMessage(): string
+    {
+        return 'Useful information for production';
+    }
+
+    protected function extractRequestData(Request $request): array
+    {
+        $url = $request->url();
+        $input = $request->all();
```

Comment:
I recommend for production to avoid writing of input, could be critical as it can contain the passwords in it.

Reply:
done in the 3-rd commit